addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -193,19 +299,30 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * @private
+         */
+        _discardThreadChangeDescription() {
+            this.update({
+                isEditingThreadDescription: false,
+                pendingThreadDescription: clear(),
+            });
+        }
+
         /**
          * @private
          * @param {MouseEvent} ev
          */
         _onClickCaptureGlobal(ev) {
-            if (!this.threadNameInputRef) {
-                return;
-            }
-            if (this.threadNameInputRef.el && this.threadNameInputRef.el.contains(ev.target)) {
-                return;
-            }
             if (this.isEditingThreadName) {
-                this._applyThreadRename();
+                if (this.threadNameInputRef.el && !this.threadNameInputRef.el.contains(ev.target)) {
+                    this._applyThreadRename();
+                }
+            }
+            if (this.isEditingThreadDescription) {
+                if (this.threadDescriptionInputRef.el && !this.threadDescriptionInputRef.el.contains(ev.target)) {
+                    this._applyThreadChangeDescription();
+                }
```

Comment:
no need to nest those conditions, just have them all together with a multi-line `if`